internal/constants/constants.go

```diff
@@ -154,6 +154,12 @@ const (
 	MiniConstellationUID = "mini"
 	// TerraformLogFile is the file name of the Terraform log file.
 	TerraformLogFile = "terraform.log"
+	// TerraformUpgradePlanFile is the file name of the zipfile created by Terraform plan for Constellation upgrades.
+	TerraformUpgradePlanFile = "plan.zip"
+	// TerraformUpgradeWorkingDir is the directory name for the Terraform workspace being used in an update.
+	TerraformUpgradeWorkingDir = "constellation-upgrade-terraform"
```

Comment:
maybe make this a subfolder of `constellation-upgrade`? 
During service upgrades we are also creating a local folder structure. See [here](https://github.com/edgelesssys/constellation/blob/main/cli/internal/helm/backup.go#LL30C32-L30C32).